apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx

```diff
@@ -91,14 +106,23 @@ export default class AdminCohortView extends React.Component {
       // Make sure we include all form_data keys that appear on any row:
       Object.keys(row.form_data).forEach(formDataHeader => {
         if (!headers[formDataHeader]) {
-          // Use the raw formData key as the column header
-          headers[formDataHeader] = formDataHeader;
+          // Convert formData key to more readable format, use it as the column header
+          headers[formDataHeader] = _.startCase(formDataHeader);
         }
       });
 
       return {...row, ...row.form_data};
     });
 
+    // Sanitize string content before exporting it to CSV
+    filteredCohortWithFormData.forEach(row => {
+      Object.keys(row).forEach(key => {
+        if (_.isString(row[key])) {
```

Comment:
Why not `typeof row[key] === 'string'`?

Reply:
I learn that idea from here https://stackoverflow.com/a/9436948; it seems like the safe way is to do `typeof myVar === 'string' || myVar instanceof String`. I'm not fully confident with Javascript yet to know if there is any other corner case, so I opt to use a library instead. What do you think?

Reply:
Got it - using the helper is fine.  In practice I think it's so rare for us to use `new String()` that the `instanceof` call isn't needed, but no complaints from me about covering all possibilities!